tools/check-parser-uptodate-or-warn.sh

```diff
@@ -16,10 +16,7 @@
 #**************************************************************************
 
 # stop early if we are not on a development version
-if test -z "$(cat VERSION | grep '+dev')"
-then
-    exit 0
-fi
+grep -Fq '+dev' VERSION || exit 0
```

Comment:
How portable is this change to different `grep` implementations? I'm worried about using anything more than the simplest `grep` command.

Reply:
I already checked - `-F` and `-q` are both POSIX, as is the combination of them to `-Fq`!

Reply:
(for the others, the `-F` is overkill, for this one it seems sensible in case some naughty `grep` treats `+` as magic without enabling EREs)